app/app.py

```diff
@@ -213,6 +236,22 @@ def query_llm(prediction_endpoint: str, messages: Dict[str, str]) -> str:
     return summary_txt
 
 
+def post_response_to_metric_service(metric_service_endpoint: str, response: str) -> str:
+    """Send the LLM's response to the metric service for readability computation using a POST request.
+
+    Args:
+        metric_service_endpoint (str): the metric service endpoint where the readability is computed
+        response (str): the response produced by the LLM
+
+    Returns:
+        str: the message received in response to the POST request
+    """
+    response_dict = {"response": response}
+    result = requests.post(url=metric_service_endpoint, json=response_dict)
+
+    return result.text
```

Comment:
Could just return the `result` here, rather than an ambiguous string? Then later log the `status_code` and `text`